Vault2GitLib/Processor.cs

```diff
@@ -293,7 +293,7 @@ private int vaultPopulateInfo(string repoPath, IDictionary<long, VaultVersionInf
             foreach (var i in ServerOperations.ProcessCommandVersionHistory(repoPath,
                                                                             1,
                                                                             VaultDateTime.Parse("2000-01-01"),
-                                                                            VaultDateTime.Parse("2020-01-01"),
+                                                                            VaultDateTime.Parse("2120-01-01"),

```

Comment:
What versions of Vault did you check this parsing with? Just confirming there is no max supported date in vault dlls themselves.

Reply:
To avoid extensive manual testing on all versions of Vault, please use currentDate + 2 for the second argument.

Reply:
I will work on this in a few days.